examples/blog-starter/lib/api.js

```diff
@@ -35,5 +35,11 @@ export function getPostBySlug(slug, fields = []) {
 
 export function getAllPosts(fields = []) {
   const slugs = getPostSlugs()
-  return slugs.map(slug => getPostBySlug(slug, fields))
+  const posts = slugs.map(slug => getPostBySlug(slug, fields))
+  // sort posts by date in descending order
+  posts.sort(function(a, b) {
+    var dateA = new Date(a.date), dateB = new Date(b.date);
```

Comment:
Let's switch from `var` to `const`, and also, can we make the sort fn by comparing the timestamps instead? without having to create a `Date` object first.

Reply:
Thanks for taking a look and suggestions @lfades
I've simplified the comparison logic https://github.com/zeit/next.js/pull/12303/commits/1e19887403c13999d7e2ab88e6ab0abeb14c32cf#diff-61ee1af0e5757bed13472594eccb0a19R41
Please let me know what you think